habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/migration/AbstractTomlGroupMigration.java

```diff
@@ -0,0 +1,157 @@
+package org.technologybrewery.habushu.migration;
```

Comment:
I (Informational): refactored existing group migration to an abstract class to make it easy to add multiple